requirements.txt

```diff
@@ -0,0 +1,2 @@
+Django==1.11.5
```

Comment:
Did you execute the web app on django 1.11.5? All the syntax are okay? Everythings running good?